### PR TITLE
CAL-511 updated docs for NSILI endpoint

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Reformats documentation for NSILI endpoint to match content outline inherited from these changes to templates: https://github.com/codice/ddf/pull/4382

#### Who is reviewing it? 
@bakejeyner @austinsteffes @garrettfreibott 

#### Choose 2 committers to review/merge the PR.

@brendan-hofmann
@brjeter
@clockard
@lessarderic
@ricklarsen - Documentation
@shaundmorris

#### Any background context you want to provide?

These changes are primarily structural. Future rewrites of individual sections will follow on from this work.

#### What are the relevant tickets?

[CAL-551](https://codice.atlassian.net/browse/CAL-551)
[DDF-04331](https://github.com/codice/ddf/pull/4382)

#### Screenshots (if appropriate)

[CAL-511-docs-revised.zip](https://github.com/codice/alliance/files/2866856/CAL-511-docs-revised.zip)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_integrating/_endpoints/nsili-endpoint.adoc [hunk 1]
```diff
@@ -1,33 +1,12 @@
...
-.. If the application is not installed, see <<{reference-prefix}installing_nsili,Installing NSILI>> to install.
-. Select *Configuration*.
-. Select *NSILI Endpoint Configuration*.
-
-See <<{reference-prefix}org.codice.alliance.nsili.endpoint,NSILI Endpoint configurations>> for all possible configurations.
-
-===== NSILI Endpoint URL
-
-The NSILI Endpoint is accessible from \${secure_url}/services/nsili.
+The NSILI Endpoint enables a client to search by http://www.gwg.nga.mil/documents/ntb/STANAG_4559_ed2.pdf[NSILI] {external-link} compliant metadata.
```
Comment (brjeter) [inline distribution/docs/src/main/resources/content/_integrating/_endpoints/nsili-endpoint.adoc]: ❓ I assumed we're going to fill this section out quite a bit right?

File: distribution/docs/src/main/resources/content/_managing/_configuring/configuring-nsili-endpoint.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
+:title: Configuring NSILI Endpoint
+:type: subConfiguration
+:status: published
+:parent: Configuring Endpoints
+:summary: Configuring NSILI Endpoint
+:order: 03
+
+== {title}
```
Comment (brjeter) [inline distribution/docs/src/main/resources/content/_managing/_configuring/configuring-nsili-endpoint.adoc]: ✏️ In the zip you provided on the PR, this is rendering as "7.8.6.5. Configuring KML Endpoint"

File: distribution/docs/src/main/resources/content/_integrating/_endpoints/nsili-endpoint.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_integrating/_endpoints/nsili-endpoint.adoc]: Yes.

File: distribution/docs/src/main/resources/content/_managing/_configuring/configuring-nsili-endpoint.adoc (hunk 2 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_managing/_configuring/configuring-nsili-endpoint.adoc]: Fixed

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_managing/_configuring/configuring-nsili-endpoint.adoc]: I'm getting errors trying to upload a new zip though

**Comment by ricklarsen** [comment]
build now

**Comment by cxbot1** [comment]
Internal build has been scheduled, your results will be available at build completion.

**Comment by cxbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.codice.org/job/Alliance-Jobs/job/pr/job/Linux/1065/
:white_check_mark: JOB SUCCESS

**Comment by bakejeyner** [comment]
Hero ✔️

**Pull request merged**